lib/survey_gizmo/configuration.rb

```diff
@@ -41,22 +41,28 @@ def reset!
   end
 
   class Configuration
-    DEFAULT_API_VERSION = 'v4'
+    DEFAULT_API = :surveygizmo
+    DEFAULT_API_VERSION = 'v4'.freeze
```

Comment:
not related to this PR but I froze some of the constant that weren't frozen, let me know if you think it shouldn't